platform/linux-generic/arch/aarch64/odp/api/abi/ticketlock_inlines.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2016-2018, Linaro Limited
+ * Copyright (c) 2021, Nokia
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @file
+ *
+ * ODP Ticketlock inline functions
+ */
+#ifndef ODP_API_ABI_TICKETLOCK_INLINES_H_
+#define ODP_API_ABI_TICKETLOCK_INLINES_H_
+
+static inline uint32_t monitorll32(uint32_t *var, int mm)
```

Comment:
Since only acquire variant of the function call is used, this entire function could be removed and just the assembly line moved inside _odp_wait_until_equal_u32().